lib/livereload.js

```diff
@@ -23,111 +24,72 @@ module.exports = LiveReload;
 /**
  * @constructor
  * @param {string} projectRoot - root folder of the project
- * @param {string} platformWWW - www directories of the platforms
- * @param {Object} opts - Options to initialize LiveReload with
- * @param {Boolean} opts.openBrowser - should a browser window be launched after browserSync is initialized ? 
- * @param {Object} prepareFn - Cordova's prepare function
+ * @param {string[]} platformWWWs - www directories of the platforms
+ * @param {Object} options - Options to initialize LiveReload with - These are to be passed to the underlying BrowserSync.
+ *                           See http://www.browsersync.io/docs/options/ for documentation
  */
-function LiveReload(projectRoot, platformWWWs, opts, prepareFn) {
-
-    if (!opts) {
-        opts = {
-            openBrowser: false
-        };
-    }
-
-    this.prepareFn = prepareFn;
-
-    // Set default options if necessary
-    this.opts = opts;
-    this.opts.openBrowser = opts.openBrowser || false;
-
+function LiveReload(projectRoot, platformWWWs, options) {
     this.projectRoot = projectRoot;
     this.startPage = helpers.GetStartPage(this.projectRoot);
     this.www_dir = path.join(this.projectRoot, 'www');
     platformWWWDirs = platformWWWs;
     this.StartServer = StartServer;
     this.StopServer = StopServer;
+    this.options = options;
+    this.browserSync = require('browser-sync').create(); // If this is exposed to users, how would they handle deps install ?
+    return this;
 };
 
 
 function StartServer() {
```

Comment:
AFAIK Methods is javascript should not be capitalized unless they are constructor methods